distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -170,6 +185,12 @@ public void beforeTest() throws Exception {
         waitForAllBundles();
         configurePDP();
         waitForHttpEndpoint(SERVICE_ROOT + "/catalog/query");
+        keystore = KeyStore.getInstance("jks");
```

Comment:
I think I originally made them to do the auth (like on line 389) but now that I look at it, it looks like they only need the paths and the password. I'll remove them.